### ISSUE TITLE
Performance on Synthia -> Cityscapes

### ISSUE THREAD

**Issue by zhijiew** [JDAI-CV/FADA#8]
Hi, thank you for your great work. I'm trying to reproduce the results of Synthia -> Cityscapes, but the performance is not as good as yours. Here are 2 questions:

In your pretrained s2c_adv.pth, the mIoU is around 38% on the validation set. Did you use this checkpoint to generate pseudo labels for self distill training and achieve mIoU 45%?

I found that in the self distill [config file](https://github.com/JDAI-CV/FADA/blob/master/configs/deeplabv2_r101_tgt_self_distill.yaml#L4) of GTA->Cityscapes, you set the freeze batch norm to False while for the [same step](https://github.com/JDAI-CV/FADA/blob/master/configs/deeplabv2_r101_tgt_synthia_self_distill.yaml#L4) of Synthia -> Cityscapes, the setting of freeze batch norm is True, why do we need this change? Does it affect the final performance?

If possible, could you provide a script for reproducing the synthia->cityscapes result, just like [train_with_sd.sh](https://github.com/JDAI-CV/FADA/blob/master/train_with_sd.sh)?

**Comment by krumo** [comment]
Hi, thanks for your interest!

I used s2c_adv.pth to generate pseudo labels and obtained the performance reported in the paper. As mentioned in #7 , you should use configs/deeplabv2_r101_adv_synthia.yaml to evaluate the checkpoint and you are supposed to obtain about 40% mIoU at 16 classes on validation set.

We made this change because we found unfreeeze BN for Synthia -> Cityscapes would harm the performance a little bit.

As all necessary config files have been provided, all you need to do is just to replace the .yaml files with corresponding *_synthia_*.yaml. I believe it would be very easy to change the script for Synthia -> Cityscapes.
